src/features/operator/Market/MarketWithdraw/index.tsx

```diff
@@ -118,6 +129,170 @@ export const MarketWithdraw = (): JSX.Element => {
     }
   };
 
+  const baseToPreferredCurrency = React.useMemo(() => {
+    // Wait for prices to load (or error out) before displaying conversions
+    if (isLoadingPrices || isErrorPrices) {
+      return '';
+    }
+
+    let amount = Big(0);
+    try {
+      amount = Big(balanceBaseAmount);
+    } catch {
+      // ignore user typos, just leave the amount as 0
+    }
+    const assetId = selectedMarket.baseAsset?.asset_id || '';
+    const assetName = selectedMarket.baseAsset?.ticker || 'unknown';
```

Comment:
Rename `assetName` to `assetTicker`. I would personally not create a new variable but it's more a matter of coding style.